python/phonenumbers/phonenumberutil.py

```diff
@@ -552,6 +552,7 @@ def _copy_number_format(other):
 
 
 def _extract_possible_number(number):
+    # type: (str) -> None
```

Comment:
I'd naively expect this to be `(str) -> str` rather than `None` &ndash; maybe I need to learn more about Python type annotations&hellip;

Reply:
I believe you are correct. I was attempting to follow the convention laid out in examples provided in PEP 484 [here](https://www.python.org/dev/peps/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code). The code they include is as follows:
```
def embezzle(self, account, funds=1000000, *fake_receipts):
    # type: (str, int, *str) -> None
    """Embezzle funds from account using fake receipts."""
    <code goes here>
```
I was assuming they were using none for some sort of compatibility reason, but digging further into the documents, this is meant to specify the return type. I will correct this and submit another PR.